s5_subwords/local/subword_methods/train_morfessor.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#Implematation of the morfessor subword segmentations
+#The tool is avalible here https://github.com/Waino/morfessor-emprune
+#Author David Erik Mollberg
+
+#if [ $# -ne 3 ]; then
+#	error "Usage: $0 <traning-corpus> <model-path>"
+#fi
+
+corpus=$1
+dir=$2
+testdata=$3
+
+mkdir -p $dir
+cut -d' ' -f2- $corpus | sed 's/ /\n/g' > $dir/corpus 
+
+corpus=$dir/corpus
+
+# Create 1M substring seed lexicon direct from a pretokenized corpus
+freq_substr.py --lex-size 1000000 < $corpus > $dir/freq_substr.1M
```

Comment:
I don't find this python file or where the morfessor executables come from

Reply:
Ah, that wasn't supposed to go with this commit. I will be adding the method later.